rsa/cli.py

```diff
@@ -178,7 +178,7 @@ def parse_cli(self) -> typing.Tuple[optparse.Values, typing.List[str]]:
     def read_key(self, filename: str, keyform: str) -> rsa.key.AbstractKey:
         """Reads a public or private key."""
 
-        print("Reading %s key from %s" % (self.keyname, filename), file=sys.stderr)
```

Comment:
What's the reason for replacing `%`-formatting? Is that being deprecated in 3.12? I don't see anything about this in https://docs.python.org/3.12/whatsnew/3.12.html#deprecated but I might overlook something. If this change is not due to Python version changes, IMO it should not be part of this PR.

Reply:
No, they're still supported. I just ran https://github.com/asottile/pyupgrade and let it do its thing.

I'll drop these changes from the PR.

Or we could convert to f-strings instead?

Reply:
> No, they're still supported. I just ran https://github.com/asottile/pyupgrade and let it do its thing.
> 
> I'll drop these changes from the PR.

Thanks.

> Or we could convert to f-strings instead?

Even if we were to do that, it should be in a separate PR.